.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,19 @@
+- id: buf-breaking
+  name: buf breaking
+  language: golang
+  entry: buf breaking proto
+  # match files ending in .proto, .protobuf, or .pb
```

Comment:
what are `.protobuf` and `.pb` for? these aren't protobuf file extensions

Reply:
I've seen those extensions used for proto files in previous companies. Sounds like buf prefers to be opinionated about the extension here, so I updated it to only include the `.proto` extension.

this is overridable by end users so it will not preclude repos with other extensions.

Reply:
`.proto` is the canonical Protobuf source file extensions, and is the only extension I've seen in the wild, and the only one recognized by `buf`.